selftests/unit/test_utils_partition.py

```diff
@@ -30,6 +31,8 @@ class TestPartition(unittest.TestCase):
     Unit tests for avocado.utils.partition
     """
 
+    @unittest.skipIf(sys.platform.startswith('darwin'),
```

Comment:
How about only checking for `/proc/mounts` location? That would work for all kind of OSs, not just mac.